requirements/prod.in

```diff
@@ -1,3 +1,5 @@
+contextvars
```

Comment:
We should ideally make sure that `contextvars` is only a requirement for the versions of Python that don't have it in the stdlib